docs/index.asciidoc

```diff
@@ -337,6 +337,33 @@ selector definitions must be double quoted in the Logstash configuration file, a
 single quoted, and numeric property values not quoted at all.
 
 
+===== Failed to Create Event with MissingConverterException
+
+Messages from certain JMS providers may contain headers or properties that Logstash cannot interpret, which can lead to
+error messages such as
+
+
+[source,txt]
+-----
+[2019-11-25T08:04:28,769][ERROR][logstash.inputs.jms      ] Failed to create event {:message=>Java::ComSolacesystemsJmsMessage::SolTextMessage: ...
+Attributes: {:jms_correlation_id=>"xxxx", :jms_delivery_mode_sym=>:non_persistent, :jms_destination=>"destination", :jms_expiration=>0, :jms_message_id=>"xxxxxx", :jms_priority=>0, :jms_redelivered=>false, :jms_reply_to=>#<Java::ComSolacesystemsJmsImpl::SolTopicImpl:0xdeadbeef>, :jms_timestamp=>1574669008862, :jms_type=>nil}
+Properties: nil, :exception=>org.logstash.MissingConverterException: Missing Converter handling for full class name=com.solacesystems.jms.impl.SolTopicImpl, simple name=SolTopicImpl, :backtrace=>["org.logstash.Valuefier.fallbackConvert(Valuefier.java:98)..."]}
+:exception=>org.logstash.MissingConverterException: Missing Converter handling for full class name=com.solacesystems.jms.impl.SolTopicImpl
+-----
+
+To get around this, use the `skip_headers` or `skip_properties` configuration setting to avoid attempting to process
+the offending header or property in the message.
+
+For example, in the example shown above, there is the attribute
+`:jms_reply_to=>#<Java::ComSolacesystemsJmsImpl::SolTopicImpl:0xdeadbeef>`, which is causing the
+`MissingConverterException`. To avoid this error, the configuration should include the following line:
```

Comment:
Possible reword for easier parsing: 
```
In the example shown above, this attribute is causing the
`MissingConverterException:

`jms_reply_to=>#<Java::ComSolacesystemsJmsImpl::SolTopicImpl:0xdeadbeef>` 

```

It renders  like this:
<img width="1009" alt="Screen Shot 2019-12-04 at 2 29 05 PM" src="https://user-images.githubusercontent.com/35154725/70174263-8af44a00-16a2-11ea-9d8a-94d008536a20.png">

If you think this is a good approach, you could also consider formatting the attribute as a chunk of ruby code, since that's what it is.

Reply:
@karenzone  The `jms_reply_to=>#<Java::ComSolacesystemsJmsImpl::SolTopicImpl:0xdeadbeef>` referenced above is a snipped taken from a log file, rather than configuration or code - do you think it should still be formatted as you suggested

Reply:
Even though it's a snippet, it's a long, wrapping snippet.  I think it's easier to read and understand if we call it out at the end.  If you disagree, I don't have strong objections to leaving it as is.